na_protein_fastq.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import modules.dna_rna_tools as dna_rna_tools
+import modules.protein_tools as protein_tools
+import modules.fastq_tool as fastq_tool
```

Comment:
```suggestion
import modules.fastqc_tool as fastq_tool
```
Из-за неправильного имени модуля все сразу падало